pkg/tsdb/azuremonitor/loganalytics/utils.go

```diff
@@ -39,7 +44,7 @@ func AddConfigLinks(frame data.Frame, dl string, title *string) data.Frame {
 		URL:         dl,
 	}
 
-	frame = AddCustomDataLink(frame, deepLink)
+	frame = AddCustomDataLink(frame, deepLink, false)
```

Comment:
not a blocking issue, but it would read easier at the callsites if there were some named variables like:
```
const SingleField bool = true
const MultipleField bool = false
```